src/app/models/user-profile.ts

```diff
@@ -0,0 +1,15 @@
+export class UserProfile {
+  id?: number;
```

Comment:
This is so that we can pass updated user info without the password which needs to be validated, unless we just want to pass in the a placeholder password that fits the validation